spec/lib/guard/notifier_spec.rb

```diff
@@ -41,4 +41,42 @@
       end
     end
   end
+
+  describe "backward compatibility" do
+    let(:notifier) { instance_double("Notiffany::Notifier") }
+
+    before do
+      allow(Notiffany::Notifier).to receive(:new).and_return(notifier)
+    end
+
+    after do
+      Guard::Notifier.instance_variable_set(:@notifier, nil)
+    end
+
+    context "multiple parameters" do
+      before do
+        subject.connect
```

Comment:
Both single and multiple param cases can share this "connect" line - no need to duplicate among these tests.